pysnooper/pycompat.py

```diff
@@ -80,3 +80,19 @@ def time_isoformat(time, timespec='microseconds'):
         return result
 
 
+if sys.version_info[:2] >= (3, 7):
+    time_fromisoformat = datetime_module.time.fromisoformat
+else:
+    def time_fromisoformat(isoformat_str):
+        hour, minute, second, microsecond = map(
+            int,
+            isoformat_str.replace('.', ':').split(':'))
+        return datetime_module.time(hour, minute, second, microsecond)
```

Comment:
@cool-RR what do you think of

```suggestion
        parts = map(int, isoformat_str.replace('.', ':').split(':'))
        return datetime_module.time(*parts)
```

Reply:
Let's leave this for later. I want to get the pr merged before the author gets too frustrated, and then improve it later. This also goes for all your other comments, except the frame dict.